poc/shared_infrastructure/main.tf

```diff
@@ -31,6 +31,42 @@ resource "azurerm_storage_account" "sa" {
   account_replication_type = "LRS"
 }
 
+resource "azurerm_cosmosdb_account" "db" {
+  name                = var.cosmosdb_account_name
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+  offer_type          = "Standard"
+  kind                = "GlobalDocumentDB"
+
+  enable_automatic_failover = true
+
+  # capabilities {
+  #   name = "EnableAggregationPipeline"
+  # }
+
+  # capabilities {
+  #   name = "mongoEnableDocLevelTTL"
+  # }
+
+  # capabilities {
+  #   name = "MongoDBv3.4"
+  # }
```

Comment:
Do we intend to eventually use these capabilities? Can we remove them? Looks like they match the examples at https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account?

Reply:
Yes, I'll remove.